versions/3.1.1.md

```diff
@@ -211,13 +213,15 @@ Types that are not accompanied by a `format` property follow the type definition
 
 The formats defined by the OAS are:
 
-| [`type`](#data-types) | [`format`](#data-type-format) | Comments                     |
-| -------------------- | --------------------------- | ---------------------------- |
-| `integer`            | `int32`                     | signed 32 bits               |
-| `integer`            | `int64`                     | signed 64 bits (a.k.a long)  |
-| `number`             | `float`                     |                              |
-| `number`             | `double`                    |                              |
-| `string`             | `password`                  | A hint to obscure the value. |
+| [Data Model Type](#data-types) | [`format`](#data-type-format) | Comments                     |
```

Comment:
I know you like the term "Data Model Type", but for me the light bulb went on when I read in the JSON Schema spec "Instance Type" -- the type of the instance being processed. Could we use that term instead?

Reply:
@mikekistler do most people understand the term "instance"?  Would perhaps "JSON Data Type" work better and be less jargon-y?

Reply:
Yes, please, make it "JSON Data Type".